### ISSUE TITLE
Protection Warrior - Shield Bash PVP Talent doesnt appear in rotation

### ISSUE THREAD

**Issue by Aum280** [herotc/hero-rotation#335]
Protection Warrior's PVP Talent Shiled Bash does not appear in the rotation.

**Comment by chrislopez24** [comment]
Shield bash is not in the rotation. The addon is meant for pve usage, not pvp.

Feel free to open to contribute and open a pull request

**Comment by jo32** [comment]
Is the rotations of warrior(Fury & Protection) up to date since last commit is done months ago?

**Comment by Aum280** [comment]
I dont think so. Sudden Death procs don't show up, some times Slam procs dont show up etc.

I think it can be optimized. And trashcans from Simcraft dont even support Protection spec.

**Comment by jo32** [comment]
I do hope simc and hero-rotation can support Protection Warrior as this spec is becoming fun in 8.1.
